### PR TITLE
Create ISSUE_TEMPLATE.md

### PR DESCRIPTION
**Pull request by arichiv-usds**
#151

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+## Screenshot/Mock <!-- Pick one -->
+
+<!-- Insert an image of the bug or sketch of the feature if relevent` -->
+
+<!--
+Please:
+* Clearly and concisely name your issue
+* Search for duplicate issues https://github.com/deptofdefense/code.mil/issues
+* Split different bugs/features into different issues
+* Add tags to the issue
```
Comment (jordangov) [inline .github/ISSUE_TEMPLATE.md]: I don't know that people can do this that aren't collaborators, so let's take this off. Otherwise it looks good!

Comment (arichiv-usds) [reply .github/ISSUE_TEMPLATE.md]: I'm not sure what you mean?

Comment (jordangov) [reply .github/ISSUE_TEMPLATE.md]: Labels are a collaborator only thing, so random GH users can't add that

Comment (arichiv-usds) [reply .github/ISSUE_TEMPLATE.md]: They can't apply labels?

Comment (jordangov) [reply .github/ISSUE_TEMPLATE.md]: Nope. Try creating an issue on anyone else's repo, you can't add a label.

**Pull request merged**